proxy/src/main/java/com/wavefront/agent/logsharvesting/FlushProcessor.java

```diff
@@ -32,7 +40,27 @@ public void processCounter(MetricName name, Counter counter, FlushProcessorConte
 
   @Override
   public void processHistogram(MetricName name, Histogram histogram, FlushProcessorContext context) throws Exception {
-    throw new NotImplementedException();
+    // HACK. Convert histogram to/from JSON using wavefront libraries to explode the yammer histogram
+    // into canonical wavefront gauges, e.g. foo.count, foo.min, foo.max, ...
+    List<ReportPoint> reportPoints = Lists.newLinkedList();
+    JsonNode jsonNode = JsonMetricsGenerator.generateJsonMetrics(new SingleMetricRegistry(histogram),
```

Comment:
46-50 is not ideal, I wouldn't go to JSON and back. If we are missing logic from the various yammer Metrics to ReportPoint, we should probably write it.